2-Webpack/src/webpackTodoList.js

```diff
@@ -0,0 +1,176 @@
+const todoListNameLocalStorage = "My Todo List";
+
+(function() {
+    const createCheck = (todo) => {
+        let checkDiv = document.createElement('input');
+        checkDiv.type = 'checkbox'
+        const checkDivClassName = "checked"
+        checkDiv.checked = todo.checked
+        checkDiv.classList.add(checkDivClassName)
+        checkDiv.addEventListener('click', function() {handleCheckedClick(todo.todoId)})
+        return checkDiv
+    }
+    
+    const createButton = (type, todoId) => {
+        let button = document.createElement('button');
+        const buttonClassName = `${type}-button`
+        button.classList.add(buttonClassName)
+        button.appendChild(createIcon(type, todoId))
+        button.addEventListener('click', function() {handleButtonClick(type, todoId)})
+        return button
+    } 
+    const createIcon = (type, id)  => {
+        let icon = document.createElement('i');
+        let iconClassName = ""
+        let iconId = ``
+        switch(type) {
+            case 'delete':
+                iconClassName = "fas fa-trash";
+                iconId = `icon-delete-${id}`
+                break;
+            case 'edit':
+                iconClassName = "fas fa-edit";
+                iconId = `icon-edit-${id}`
+                break;
+            case 'check':
+                iconClassName = "fa-check";
+                iconId = `icon-check-${id}`
+            default:
+                // none
+        }
+        icon.className = iconClassName;
+        icon.id = iconId
+        return icon
+    }
+    
+    
+    function handleButtonClick(type, todoId) {
+        switch(type) {
+            case 'delete':
+                handleDeleteTodo(todoId)
+                break;
+            case 'edit':
+                handleEditTodo(todoId)
+                break;
+            default:
+                // none
+        }
+    }   
+    
+    function handleCheckedClick(todoId) {
+        const todo = document.getElementById(`todo-item-${todoId}`)
+        todo.classList.toggle('completed')
+        let todos = loadTodoListFromLS()
```

Comment:
probably the most important comment here:
this file is getting huge and functions are starting to convolute with each other, it's about time to start separating into different modules, for example have the logic that accesses the `localStorage` in separate file